<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1493</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_achieve">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QVideoWidget" name="wg_videos" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>850</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QWidget" name="widget_7" native="true">
             <widget class="QDial" name="dl_pressure_upper">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>430</y>
                <width>50</width>
                <height>64</height>
               </rect>
              </property>
             </widget>
             <widget class="QDial" name="dl_pressure_lower">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>430</y>
                <width>50</width>
                <height>64</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image: url(.\\image\\dail_press.png);
</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pb_start_button">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>520</y>
                <width>75</width>
                <height>24</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>开始实验</string>
              </property>
             </widget>
             <widget class="QLabel" name="lb_press_lower">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>40</y>
                <width>121</width>
                <height>381</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>:/image/press.png</pixmap>
              </property>
             </widget>
             <widget class="QLabel" name="lb_press_upper">
              <property name="geometry">
               <rect>
                <x>190</x>
                <y>40</y>
                <width>121</width>
                <height>381</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>:/image/press.png</pixmap>
              </property>
             </widget>
             <widget class="QLabel" name="lb_move_slider_press_lower">
              <property name="geometry">
               <rect>
                <x>233</x>
                <y>310</y>
                <width>41</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>:/image/slider_press.png</pixmap>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <widget class="QDial" name="dl_frequency_ctrl">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>240</y>
                <width>50</width>
                <height>64</height>
               </rect>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QPushButton" name="pb_total">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>360</y>
                <width>75</width>
                <height>61</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{border:0px;}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QPushButton" name="pb_mixer">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>360</y>
                <width>75</width>
                <height>61</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{border:0px;}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QPushButton" name="pb_press">
              <property name="geometry">
               <rect>
                <x>360</x>
                <y>360</y>
                <width>75</width>
                <height>61</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{border:0px;}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>450</y>
                <width>53</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>总电源</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>450</y>
                <width>53</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>电机</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>380</x>
                <y>450</y>
                <width>53</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>压力</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>20</y>
                <width>431</width>
                <height>221</height>
               </rect>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_start">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="StartVideoWidget" name="wg_start_video" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1493</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StartVideoWidget</class>
   <extends>QWidget</extends>
   <header>startvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
